main.tf

```
@@ -29,4 +29,21 @@ module "zone" {
 
   name = var.name
   tags = var.tags
+  vpc  = var.vpc
```

Making things plural to support multiple vpc blocks

```suggestion
  vpcs  = var.vpcs
```